bytes/bytes/Extensions/Collection+Randomizing.swift

```diff
@@ -13,12 +13,12 @@ extension NSArray {
     /// It uses the Fisher-Yates algorithm.
     ///
     /// [source](https://gist.github.com/natecook1000/0ac03efe07f647b46dae)
-    public func shuffled() -> NSArray {
+    @objc public func shuffled() -> NSArray {
         let array = self as Array
         return array.shuffled() as NSArray
     }
     /// Returns a random element of this array
-    public var random: Element? {
+    @objc public var random: Element? {
```

Comment:
To be inline with the other changes in this PR, should we make this extension public and then remove the access levels on the properties themselves?

Reply:
This is probably a question for all the other extensions that we have 😊

Reply:
Good point. Changed the other extensions to have the same way of using access levels.